editor/editor_help.cpp

```diff
@@ -1925,6 +1927,24 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt) {
 			pos = brk_end + 1;
 			tag_stack.push_front("font");
 
+		} else if (tag == "note") {
+			// Note block.
+			p_rt->push_color(note_color);
+			p_rt->push_font(doc_bold_font);
+			p_rt->add_text(TTR("Note:") + " ");
+			p_rt->pop();
+			pos = brk_end + 1;
+			tag_stack.push_front(tag);
+
+		} else if (tag == "warning") {
+			// Warning block.
+			p_rt->push_color(warning_color);
```

Comment:
I'm not sure why it works correctly, but we should probably add a special case for note/warning blocks, like we do for code blocks, and set a flag. So when we pop the tag from the stack (around line 1768 as of your PR), we properly pop the color. 

I think it currently works without visible issues because a new color is pushed somewhere, but I don't think that we correctly handle the extra color here.